src/lib.rs

```diff
@@ -580,6 +577,16 @@ fn index_module<T: SourceContainer>(
         all_annotations.import(annotations);
         all_literals.import(string_literals);
     }
+    // don't validate includes
+    for (_, unit) in includes_units.into_iter() {
```

Comment:
I think instead of looking for include, we should just ignore any unit that is external (the {external} pragma should also not have its body validated)